RELEASING.md

```diff
@@ -2,82 +2,90 @@ Releasing clld
 ==============
 
 - Update translations
-```
-python setup.py extract_messages
-python setup.py update_catalog
-python setup.py compile_catalog
+```shell
+$ python setup.py extract_messages
+$ python setup.py update_catalog
+$ python setup.py compile_catalog
```

Comment:
I deliberately left out the shell prompt to make copy&paste of these lines easier.

Reply:
Makes sense, removing.